src/Umbraco.Web.UI/umbraco/config/lang/nl.xml

```diff
@@ -1146,7 +1153,12 @@ Echter, Runway biedt een gemakkelijke basis om je snel op weg te helpen. Als je
     <key alias="thisEditorUpdateSettings">die gebruik maken van deze editor zullen geupdate worden met deze nieuwe instellingen</key>
 
     <key alias="memberCanEdit">Lid kan bewerken</key>
+    <key alias="memberCanEditDescription">Toestaan dat deze eigenschap kan worden gewijzigd door het lid op zijn profiel pagina.</key>
+    <key alias="isSensitiveData">Omvat gevoelige gegevens</key>
+    <key alias="isSensitiveDataDescription">Verberg deze eigenschap voor de content editor dat geen toegang heeft tot het bekijken van gevoelige informatie.</key>

```

Comment:
Shouldn't dat be die

Reply:
@dawoe die geen toegang heeft?

Reply:
yep.."Verberg deze eigenschap voor de content editor die geen..."